core/src/test/java/org/apache/iceberg/util/TestSnapshotUtil.java

```diff
@@ -60,12 +62,23 @@ public class TestSnapshotUtil {
           .withRecordCount(1)
           .build();
 
-  private long snapshotAId;
+  private long snapshotBaseTimestamp;
+  private long snapshotBaseId;
+  private long snapshotBranchId;
+  private long snapshotMain1Id;
+  private long snapshotMain2Id;
+  private long snapshotFork0Id;
+  private long snapshotFork1Id;
+  private long snapshotFork2Id;
+
+  private Snapshot putSnapshot(String branch) {
```

Comment:
`putSnapshot` is a bit misleading, what about `appendFileTo(String branch)`?

Reply:
Will change it to this.